apps/web/src/hooks/useFindSessionQuery.ts

```
@@ -0,0 +1,51 @@
+import { reviver } from '@douglasneuroinformatics/libjs';
```

ESLint

[error] 'reviver' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)